vassal-app/src/main/java/VASSAL/build/module/map/SetupStack.java

```diff
@@ -231,12 +231,12 @@ public void setup(boolean gameStarting) {
         p.translate(r.x, r.y);
       }
       if (placeNonStackingSeparately()) {
-        for (int i=0;i<s.getPieceCount();++i) {
+        for (int i=0; i<s.getPieceCount(); ++i) {
```

Comment:
Spacing around `=`, `<` here.

Reply:
Will be checked by rule "WhitespaceAround" that we will enable later on. For now we have enabled "WhitespaceAfter".